addons/mail/models/mail_tracking_value.py

```diff
@@ -63,11 +63,9 @@ def _create_tracking_values(self, initial_value, new_value, col_name, col_info,
 
         :return: a dict values valid for 'mail.tracking.value' creation;
         """
-        tracked = True
-
         field = self.env['ir.model.fields']._get(record._name, col_name)
         if not field:
-            return
+            raise ValueError(f'Unknown field {col_name} on model {record._name}')
```

Comment:
Don't we want to do add an exception directly on defining tracking on an invalid field in ir.model.fields? And maybe  correct the 'tracking' value on fields in the migrate script if any?

Reply:
Anyway I think this method should be defensive ... but indeed some types should not be tracked and we could add an exception in ir model field directly; however I am unsure where this should be added in this low level model, so... letting it as an exercise for the reader.